tests/components/radarr/test_calendar.py

```diff
@@ -4,13 +4,11 @@
 
 from freezegun.api import FrozenDateTimeFactory
 
-from homeassistant.components.radarr.const import DOMAIN
 from homeassistant.const import STATE_OFF, STATE_ON
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
-
-from . import setup_integration
 
+from tests.common import async_fire_time_changed
+from tests.components.radarr import setup_integration
```

Comment:
Please use relative import (would have been caught by #119279 but that's not merged yet)